### PR TITLE
Create favorite screen

### PR DESCRIPTION
**Pull request by ngochieu1297**
## [Pull request title]
Create favorite screen and Search Activity
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7523

### CODE REVIEW

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/java/com/example/music_58/ui/MainActivity.java [hunk 1]
```diff
@@ -22,6 +29,20 @@ protected void onCreate(Bundle savedInstanceState) {
         addEvent();
     }
 
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.text_search:
+                startActivity(new Intent(MainActivity.this, SearchActivity.class));
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/MainActivity.java]: convention 2.11

File: app/src/main/java/com/example/music_58/ui/MainActivity.java [hunk 2]
```diff
@@ -22,6 +29,20 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                break;
+            case R.id.image_search:
+                startActivity(new Intent(MainActivity.this, SearchActivity.class));
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/MainActivity.java]: tương tự

File: app/src/main/java/com/example/music_58/ui/MainActivity.java [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
 
 import com.example.music_58.R;
 import com.example.music_58.ui.adapter.ViewPagerAdapter;
+import com.example.music_58.ui.search.SearchActivity;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
     private TabLayout mTabLayout;
     private ViewPager mViewPager;
     private Toolbar mToolbar;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/MainActivity.java]: local variable

File: app/src/main/res/layout/item_favorite.xml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/image_favorite_song"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/text_artist"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_favorite.xml]: hardcode

File: app/src/main/res/layout/item_local_track.xml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/image_local_song"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/text_artist"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="8dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_local_track.xml]: hardcode

File: app/src/main/res/layout/item_local_track.xml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+        android:textColor="#81ffffff"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_local_track.xml]: hardcode

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 7]
```diff
@@ -9,6 +9,7 @@
         android:roundIcon="@drawable/ic_start"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".ui.search.SearchActivity"></activity>
```
Comment (NguyenMinhTuanB) [inline app/src/main/AndroidManifest.xml]: convention 3.1

File: app/src/main/java/com/example/music_58/ui/MainActivity.java [hunk 8]
```diff
@@ -22,6 +28,25 @@ protected void onCreate(Bundle savedInstanceState) {
         addEvent();
     }
 
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.text_search:
+                startActivity(getProfileIntent(this));
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/MainActivity.java]: em đang áp dụng sai convention 2.11

File: app/src/main/java/com/example/music_58/ui/MainActivity.java [hunk 9]
```diff
@@ -22,6 +28,25 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                break;
+            case R.id.image_search:
+                startActivity(getProfileIntent(this));
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/MainActivity.java]: tương tự.

File: app/src/main/java/com/example/music_58/ui/MainActivity.java (hunk 8 above)
Comment (NguyenMinhTuanB) [reply app/src/main/java/com/example/music_58/ui/MainActivity.java]: đọc kỹ lại

**Comment by ngochieu1297** [comment]
ready

**Pull request merged**